### PR TITLE
qmlformat --reformat-codebase

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for e5ec3e32d869e73585189568013ab7b98d62a5a7: [**arm64-android**](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield-dev-5415-bffaec6912a142a147135c0bdde5bf33ef3b27a8-android-arm64.apk)
<details>
<summary>Other architectures</summary>

- [arm-neon-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield-dev-5415-bffaec6912a142a147135c0bdde5bf33ef3b27a8-android-arm-neon.apk)
- [x64-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield-dev-5415-bffaec6912a142a147135c0bdde5bf33ef3b27a8-android-x64.apk)
- [x86-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield-dev-5415-bffaec6912a142a147135c0bdde5bf33ef3b27a8-android-x86.apk)
- [**all access arm64-android**](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield_all_access-dev-5415-bffaec6912a142a147135c0bdde5bf33ef3b27a8-android-arm64.apk)
- [all access arm-neon-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield_all_access-dev-5415-bffaec6912a142a147135c0bdde5bf33ef3b27a8-android-arm-neon.apk)
- [all access x64-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield_all_access-dev-5415-bffaec6912a142a147135c0bdde5bf33ef3b27a8-android-x64.apk)
- [all access x86-android](https://qfieldapks.sos-ch-dk-2.exoscale-cdn.com/ci-builds/qfield_all_access-dev-5415-bffaec6912a142a147135c0bdde5bf33ef3b27a8-android-x86.apk)
</details>

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -81,3 +81,9 @@ repos:
     files: .zenodo.json
     types: [json]
     args: ["--schemafile", "https://raw.githubusercontent.com/rouault/zenodo_json_schema/master/zenodo-json-schema.json"]
+
+
+- repo: https://github.com/nrbnlulu/qt-hooks
+  rev: v0.1.0  # current revision
```
Comment (suricactus) [inline .pre-commit-config.yaml:87]:
```suggestion
  rev: v0.1.5
```

**Review by suricactus — APPROVED** [review]
Looks good to me, checked every single line ;) .

One request to make the precondition work on latest ubuntu - please bump the version of the qml formatter action.

**Comment by nirvn** [comment]
@m-kuhn , huge thanks for this!

**Pull request merged**